src/index.html

```diff
@@ -15,8 +15,110 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
```

Comment:
right now your links navigate to the root page where there is nothing)
Instead of `/` you can add links to some anchors on the current page
```suggestion
        href="#header"
```